benchmark_runner/workloads/workloads_operations.py

```diff
@@ -373,6 +373,15 @@ def get_run_artifacts_google_drive(self):
         run_artifacts_hierarchy = self._get_run_artifacts_hierarchy(workload_name=workload)
         return self._google_drive_operation.get_drive_folder_url(folder_path=run_artifacts_hierarchy, parent_folder_id=self._google_drive_shared_drive_id)
 
+    def create_run_artifacts_google_drive(self):
+        """
+        This method creates google drive run artifacts folder path
+        :return:
+        """
+        workload = self._workload.replace('_', '-')
+        run_artifacts_hierarchy = self._get_run_artifacts_hierarchy(workload_name=workload)
+        return self._google_drive_operation.create_drive_folder_url(folder_path=run_artifacts_hierarchy, parent_folder_id=self._google_drive_shared_drive_id)
```

Comment:
This returns a value, but the caller below (at line 537) does not use the return value as it should.